modules/tinymce/src/plugins/table/main/ts/selection/SelectionTargets.ts

```diff
@@ -42,6 +42,7 @@ export interface SelectionTargets {
   readonly onSetupUnmergeable: (api: UiApi) => () => void;
   readonly resetTargets: () => void;
   readonly onSetupTableWithCaption: (api: UiToggleApi) => () => void;
+  readonly onSetupTableHeaders: () => (api: Toolbar.ToolbarToggleButtonInstanceApi) => () => void;
```

Comment:
This should not be thunked and the type is incorrect here, as it should be `UiToggleApi`. eg:
```suggestion
  readonly onSetupTableHeaders: (api: UiToggleApi) => () => void;
```